plexupdate.sh

```diff
@@ -275,7 +275,7 @@ fi
 
 # Remove any ~ or other oddness in the path we're given
 DOWNLOADDIR="$(cd ${DOWNLOADDIR// /\\ } && pwd)"
-if [ -z "${DOWNLOADDIR}" ]; then
+if [ ! -d "${DOWNLOADDIR}" ]; then
```

Comment:
"" only points to current directory for things like cd. For tests, it's treated as a literal null string which has no path equivalent.
```bash
$ [ -d "" ] || echo nope
nope
$
```

And while you're right that downloaddir **would** be blank if cd failed anyway, checking to see if a directory path is a valid directory seemed more readable to me than checking if a directory path is an empty string.